schemas/accelerator.sql

```diff
@@ -0,0 +1,209 @@
+-- This schema adds the tables necessary for Accelerator accounting as a
+-- separate record as part of the wider Cloud Accounting system.
+
+DROP TABLE IF EXISTS AcceleratorRecords;
+CREATE TABLE AcceleratorRecords (
+  UpdateTime TIMESTAMP NOT NULL DEFAULT CURRENT_TIMESTAMP ON UPDATE CURRENT_TIMESTAMP,
+  MeasurementMonth INT NOT NULL,
+  MeasurementYear INT NOT NULL,
+
+  AssociatedRecordType VARCHAR(255) NOT NULL,
+  AssociatedRecord VARCHAR(255) NOT NULL,
+
+  GlobalUserName VARCHAR(255),      
+  FQAN VARCHAR(255) NOT NULL,       
+  SiteName VARCHAR(255) NOT NULL,   
```

Comment:
`SiteName VARCHAR(255) NOT NULL` should be `SiteID INT NOT NULL` to be consistent with other forms of APEL accounting (both in the `AcceleratorRecords` and `AcceleratorSummaries` tables).

@tofu-rocketry, was the intention the accelerators would be an "add-on" to cloud or grid accounting, or its own standalone type of accounting? If it needs to be standalone - a `Sites` table and `SitesLookup` function would need to be added similar to https://github.com/apel/apel/blob/1.9.2-1/schemas/server.sql#L420